### PR TITLE
Update guessmoji stats and leaderboards

### PR DESCRIPTION
**Pull request by h-anjru**
Updated guessmoji stats and leaderboards to clear up confusion between guessmoji and cringo stats. When guessmoji was created, it was the only luck-based game (and the only way to win crimsoCOIN). This update should clear up the differences.

### CODE REVIEW

File: crimsobot/utils/games.py [hunk 1]
```diff
@@ -200,10 +200,50 @@ async def guess_luck(discord_user: DiscordUser, n: int, won: bool) -> None:
     await stats.save()
 
 
-async def guess_luck_balance(discord_user: DiscordUser) -> Tuple[float, int]:
-    stats = await GuessStatistic.get_by_discord_user(discord_user)  # type: GuessStatistic
+# async def guess_luck_balance(discord_user: DiscordUser) -> Tuple[float, int]:
+#     stats = await GuessStatistic.get_by_discord_user(discord_user)  # type: GuessStatistic
+
+#     return stats.luck_index, stats.plays
+
```
Comment (williammck) [inline crimsobot/utils/games.py:207]: nit: remove commented out code

File: crimsobot/utils/tools.py [hunk 2]
```diff
@@ -68,10 +68,12 @@ def crimbed(title: Optional[str], descr: Optional[str], thumb_name: Optional[str
         'triumph': 'https://i.imgur.com/bBXRFnO.png',
         'joy': 'https://i.imgur.com/8deo8Ak.png',
         'hug': 'https://i.imgur.com/lSPKbWf.png',
-        'think': 'https://i.imgur.com/odD9yI2.png',
+        # 'think': 'https://i.imgur.com/odD9yI2.png',
```
Comment (williammck) [inline crimsobot/utils/tools.py:71]: nit: remove commented out code

Comment (h-anjru) [reply crimsobot/utils/tools.py:71]: i am going to keep that old link *for now*

File: crimsobot/utils/games.py (hunk 1 above)
Comment (h-anjru) [reply crimsobot/utils/games.py:207]: i did not intend to leave that, heck

File: crimsobot/utils/tools.py (hunk 2 above)
Comment (williammck) [reply crimsobot/utils/tools.py:71]: but it's in the git history! you don't need to leave it! :D

**Pull request merged**